MayssaraOmar/assignment3/Word search/src/main.cpp

```diff
@@ -0,0 +1,14 @@
+//============================================================================
+//ASSUMPTIONS
+// (dictionary) class and (get_words_from_grid) method are case sensitive
+//============================================================================
+
+#include <iostream>
+#include "test_dictionary.h"
+#include "test_wordSearch.h"
```

Comment:
Optional: it is often the case that tests are put in the same file as the main function that invoked them.